### ISSUE TITLE
Comedy Central @Midnight no longer working

### ISSUE THREAD

**Issue by augur42** [rg3/youtube-dl#8229]
Two different sets of errors, I was completely unable to get the January 11th episode to play in any browser. I was able to get the January 12th episode to play in a browser using a vpn. I was able to download both episodes using jdownloader2 beta.
```
H:\ytd>youtube-dl.exe --verbose http://www.cc.com/full-episodes/eeh6r4/-midnight
-with-chris-hardwick-monday--january-11--2016-season-3-ep-03045
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'--verbose', u'http://www.cc.com/full-episodes/eeh6
r4/-midnight-with-chris-hardwick-monday--january-11--2016-season-3-ep-03045']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2016.01.09
[debug] Python version 2.7.10 - Windows-7-6.1.7601-SP1
[debug] exe versions: none
[debug] Proxy map: {}
[ComedyCentral] -midnight-with-chris-hardwick-monday--january-11--2016-season-3-
ep-03045: Downloading webpage
[ComedyCentral] 02f09cbc-04bc-45aa-8c51-6819f2fc97dc: Downloading info
ERROR: Unable to download XML: HTTP Error 500: Internal Server Error (caused by
HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you
are using the latest version; type  youtube-dl -U  to update. Be sure to call yo
utube-dl with the --verbose flag and include its complete output.
  File "youtube_dl\extractor\common.pyo", line 351, in _request_webpage
  File "youtube_dl\YoutubeDL.pyo", line 1890, in urlopen
  File "urllib2.pyo", line 437, in open
  File "urllib2.pyo", line 550, in http_response
  File "urllib2.pyo", line 469, in error
  File "urllib2.pyo", line 409, in _call_chain
  File "urllib2.pyo", line 656, in http_error_302
  File "urllib2.pyo", line 437, in open
  File "urllib2.pyo", line 550, in http_response
  File "urllib2.pyo", line 475, in error
  File "urllib2.pyo", line 409, in _call_chain
  File "urllib2.pyo", line 558, in http_error_default


H:\ytd>youtube-dl.exe --verbose http://www.cc.com/full-episodes/v43jou/-midnight
-with-chris-hardwick-tuesday--january-12--2016-season-3-ep-03046
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'--verbose', u'http://www.cc.com/full-episodes/v43j
ou/-midnight-with-chris-hardwick-tuesday--january-12--2016-season-3-ep-03046']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2016.01.09
[debug] Python version 2.7.10 - Windows-7-6.1.7601-SP1
[debug] exe versions: none
[debug] Proxy map: {}
[ComedyCentral] -midnight-with-chris-hardwick-tuesday--january-12--2016-season-3
-ep-03046: Downloading webpage
[ComedyCentral] edcce418-07f3-4b84-9d6a-86e520976306: Downloading info
Traceback (most recent call last):
  File "__main__.py", line 19, in <module>
  File "youtube_dl\__init__.pyo", line 410, in main
  File "youtube_dl\__init__.pyo", line 400, in _real_main
  File "youtube_dl\YoutubeDL.pyo", line 1677, in download
  File "youtube_dl\YoutubeDL.pyo", line 665, in extract_info
  File "youtube_dl\extractor\common.pyo", line 312, in extract
  File "youtube_dl\extractor\mtv.pyo", line 216, in _real_extract
  File "youtube_dl\extractor\mtv.pyo", line 180, in _get_videos_info
  File "youtube_dl\extractor\mtv.pyo", line 185, in _get_videos_info_from_url
  File "youtube_dl\extractor\common.pyo", line 487, in _download_xml
  File "youtube_dl\compat.pyo", line 248, in compat_etree_fromstring
  File "youtube_dl\compat.pyo", line 238, in _XML
  File "xml\etree\ElementTree.pyo", line 1654, in close
  File "xml\etree\ElementTree.pyo", line 1506, in _raiseerror
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
```
I initially tried an older version 2016.1.1.0 and when that failed updated to latest windows exe.

**Comment by remitamine** [comment]
can you check again, may be it's a temporary error.
it works for me.
```
python2 __main__.py http://www.cc.com/full-episodes/eeh6r4/-midnight
[ComedyCentral] -midnight: Downloading webpage
[ComedyCentral] 02f09cbc-04bc-45aa-8c51-6819f2fc97dc: Downloading info
[ComedyCentral] f79867ce-16dc-42b9-86fc-08f7098df136: Extracting information
[ComedyCentral] f79867ce-16dc-42b9-86fc-08f7098df136: Downloading video urls
[ComedyCentral] 1dbac2fc-7a98-4f01-95c2-b4f583bdd6ee: Extracting information
[ComedyCentral] 1dbac2fc-7a98-4f01-95c2-b4f583bdd6ee: Downloading video urls
[ComedyCentral] f8386017-a47c-4916-b34f-bc59ee2a1168: Extracting information
[ComedyCentral] f8386017-a47c-4916-b34f-bc59ee2a1168: Downloading video urls
[ComedyCentral] 6e090934-be08-4e33-80fd-8b85bd4b80d3: Extracting information
[ComedyCentral] 6e090934-be08-4e33-80fd-8b85bd4b80d3: Downloading video urls
[download] Downloading playlist: None
[ComedyCentral] playlist None: Collected 4 video ids (downloading 4 of them)
[download] Downloading video 1 of 4
WARNING: ar subtitles not available for f79867ce-16dc-42b9-86fc-08f7098df136
WARNING: arME subtitles not available for f79867ce-16dc-42b9-86fc-08f7098df136
WARNING: enUS subtitles not available for f79867ce-16dc-42b9-86fc-08f7098df136
WARNING: fr subtitles not available for f79867ce-16dc-42b9-86fc-08f7098df136
[info] Writing video subtitles to: @midnight with Chris Hardwick_@midnight with Chris Hardwick 03045 Act 1 f79867ce-16dc-42b9-86fc-08f7098df136-f79867ce-16dc-42b9-86fc-08f7098df136.en.vtt
[download] Destination: @midnight with Chris Hardwick_@midnight with Chris Hardwick 03045 Act 1 f79867ce-16dc-42b9-86fc-08f7098df136-f79867ce-16dc-42b9-86fc-08f7098df136.mp4
[#d2e011 5.1MiB/252MiB(2%) CN:16 DL:205KiB ETA:20m31s]
```

**Comment by augur42** [comment]
Embarrassingly it must have been a temporary error, I know I am geoblocked on the main page but I can usually manage to get the stream from the cdn servers. I tried over a period of a few hours, guess I should have left it even longer. Interesting that jdownloader2 worked, I guess it uses a different method of parsing. I prefer youtube-dl as I can get subs.

At the time I thought it might be related to the mtv issue https://github.com/rg3/youtube-dl/issues/8169 guess not.

Apologies for wasting whomever's time, and thanks remitamine for the quick response.
